app/Jobs/Emails/Registration/PromoCodes/SponsorPromoCodeEmail.php

```diff
@@ -54,7 +54,12 @@ public function __construct
         $summit = $promo_code->getSummit();
         $payload = [];
         $sponsor = $promo_code->getSponsor();
-        $payload[IMailTemplatesConstants::sponsor_tier_name] = $sponsor->getSponsorship()->getType()->getName();
+        $sponsorships = $sponsor->getSponsorships();
```

Comment:
@romanetar could we create a method at Sponsor::getSponsorshipTiersNames() and return of array of names ?